app/models/participant_profile.rb

```diff
@@ -55,6 +49,10 @@ class ParticipantProfile < ApplicationRecord
 
   self.ignored_columns = %w[user_id]
 
+  def self.humanize_start_term(start_term)
+    start_term.to_s.split("_").map(&:capitalize).join(" ")
```

Comment:
```suggestion
    start_term.to_s.humanize
```

I don't think this belongs in the model. It's a view-layer helper, if anything!

Reply:
Very much agree and was really unhappy about adding this in. helper does not feel right either here (they never do tbh), but I didn't want to repeat the same logic in all the places it is required. It is however sort-of a view concern, so I will, with heavy heart, move this to helper